### PR TITLE
CSS changes to Create Task

### PR DESCRIPTION
**Pull request by JSiu**
I _think_ this makes the fields line up on the Create Task page, to match the way our other forms look.  Of course, with the now-usual caveat of cursing at my VM.

### CODE REVIEW

**Comment by natmote** [comment]
I'll take a look at this in a bit, but I'd like to point out that you could probably run our project on attu -- I just checked and it has virtualenv installed, which is pretty much all you need, barring any weird compatibility errors.

Since this VM thing seems to be non-temporary, you might want to look in to it.

File: fullhouse/templates/nonhousemember.html [hunk 1]
```diff
@@ -31,6 +31,7 @@ <h2 class="reg-header" align="center">Oh! It looks like you aren't a member of a
             {{ form.name }}
           </div>
 
+
```
Comment (creisman) [inline fullhouse/templates/nonhousemember.html]: Get rid of this extra line.

**Comment by JSiu** [comment]
@natmote, Conor said he would let you merge this, if you would be so good as to verify that it works.

**Comment by natmote** [comment]
This looks good. When there are errors, the bullet points are way off to the left, and the text is over on the right, so that looks ugly, but overall this is an improvement.

**Pull request merged**